modules/appeals_api/app/services/appeals_api/pdf_construction/higher_level_review/v2/structure.rb

```diff
@@ -37,10 +38,23 @@ def form_fill
             form_fields.veteran_phone_line_number => form_data.veteran_phone_line_number,
             form_fields.veteran_phone_international_number => form_data.veteran_phone_international_number,
 
-            # Section II: Claimant's ID
-            # NOT YET SUPPORTED
-
-            # Section III: Benefit Type
+            # Claimant
+            # Claimant name is filled out through autosize text box, not pdf fields
+            form_fields.claimant_middle_initial => form_data.claimant_middle_initial,
```

Comment:
`form_data.claimant.middle_initial` would be preferable here, IMO. No need to hide the claimant data behind a bunch of facade methods.

Reply:
We can use the null object pattern for the claimant object - basically you can still instantiate an instance even if it doesn't have any/all the data, and anything it can't respond to just returns a nil.

`myClaimant = Claimant.new(nil_headers, nil_claimant_data)
myClaimant.last_name # => nil`

This way, the claimant object will always be present and we won't have to do to any nil checks from the structure at all :)

Reply:
Ohhhh. Nathan mentioned this but I thought it was far more complicated than this! I will work on it.